### PR TITLE
fix: login redirect

### PR DESCRIPTION
**Pull request by melniiv**
### Closes :no_good_woman:

**[PT-1817](https://helsinkisolutionoffice.atlassian.net/browse/PT-1817)**

## Additional notes :spiral_notepad:

[PT-1817]: https://helsinkisolutionoffice.atlassian.net/browse/PT-1817?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr370.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr370.dev.hel.ninja** :rocket::rocket::rocket:

File: .env.development [hunk 1]
```diff
@@ -8,12 +8,12 @@
 VITE_APP_API_URI=https://kultus.api.test.hel.ninja/graphql
 VITE_APP_API_REPORT_URI=https://kultus.api.test.hel.ninja/reports
 VITE_APP_LINKEDEVENTS_API_URI=https://linkedevents.api.test.hel.ninja/v1
-VITE_APP_OIDC_AUDIENCES="kultus-api-dev,profile-api-test"
+VITE_APP_OIDC_AUDIENCES="kultus-api-test,profile-api-test"
```
Comment (nikomakela) [inline .env.development:11]: this file should be excluded totally from repo.

File: src/domain/app/login/LoginPage.tsx [hunk 2]
```diff
@@ -9,9 +10,10 @@ import styles from './loginPage.module.scss';
 const LoginPage = () => {
   const { t } = useTranslation();
   const { login } = useOidcClient();
+  const path = useLocation();
```
Comment (nikomakela) [inline src/domain/app/login/LoginPage.tsx:13]: Destruct the pathname would be more elegant. 🙂

**Review by nikomakela — APPROVED** [review]
The env files should not be publicly available. Otherwise OK.

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr370.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**